code-challenges.js

```diff
@@ -26,20 +26,36 @@ const fruit2 = "banana"
 
 const padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
 const padres1998WorldSeriesRuns = [6, 3, 5, 3]
+const all = (padres1984WorldSeriesRuns.concat(padres1998WorldSeriesRuns))
+
+console.log(all.length)
+
 // Expected output: 9
 
 // Pseudo code:
+// Adding the length of my first and second arrays to get my result.
 
 // --------------------2) Create the code that will reverse the letters of a string using the test variable provided below.
 
-const currentCohort = "Golf 2023"
-// Expected output: "3202 floG"
+const currentCohort = "Hotel NOT Golf 2023"
+console.log(currentCohort.split('').reverse().join(''))
+
+// Expected output: "3202 floG TON letoH"
 
 // Pseudo code:
+// Since you can't reverse a string I first converted currentCohort to an array via .split then reversed the array with the .reverse method before finally turning it back into a string for my result using the .join method.
 
 // --------------------3) Create the code that will log only the odd numbers from the array using the test variable provided below.
 
 const stockExchange = [13, 34, -4, 42, 5, -5, 10, 27, 42, 10]
+
+for (let i = 0; i < stockExchange.length; i++){
+    if (stockExchange[i] % 2 !== 0){
+        console.log(stockExchange[i])
+    }      
+}
+
 // Expected output: 13 5 -5 27
 
 // Pseudo code:
+// Honestly I don't really know how to explain it. The for loop tells the computer to perform a cycle at index 0 of the array, that index 0 is less than the length of the array and to go the length of the array in increments of 1. If the variable index divided by two does not equal 0 then print out that number.  
```

Comment:
Here's an example of good pseudo code for this code:
- start iteration using a for loop
- each iteration checks if the current value of the iteration index in the array does not have a remainder of 0 when divided by 2
- if that's the case, the value gets logged in the console, and the iteration continues

You can use this same format for all pseudo code